src/lib/emojis.js

```diff
@@ -7,6 +7,7 @@ export const emojisByName = {
   clock: '🕑',
   check: '✅',
   party_popper: '🎉',
+  full_moon_with_face: '🌝',
```

Comment:
Why is there a dangling comma here? This is not Go.

Reply:
That is due to @luan's lint rules!